Dockerfile

```diff
@@ -14,7 +14,6 @@ COPY --from=builder /busybox/cp /cp
 WORKDIR /
 
 COPY manager manager
-COPY unpack unpack
```

Comment:
You should also be able to remove the staged build components and copying of `cp` binary above. Also be sure to `rm -rf` the cmd/unpack :smiley:

Reply:
Do we need to, if we don't build it?